crates/sui-core/src/authority.rs

```diff
@@ -926,6 +927,41 @@ impl AuthorityState {
             .expect("notify_read_effects should return exactly 1 element"))
     }
 
+    fn receiving_object_transaction_dependencies(
```

Comment:
This calculates the transaction dependencies for any `Receiving` arguments. 

The actual object for the `Receiving` argument may actually no longer exist, and therefore we need to do the usual trick of "try load the object, but if it isn't there fallback to the object marker table" to get the previous transaction's digest so we can correctly add the dependency.